Assets/Shopify/UIToolkit/Shops/Generic/Classes/GenericSingleProductShop.cs

```diff
@@ -86,123 +62,19 @@ public class GenericSingleProductShop : MonoBehaviour, ISingleProductShop {
         }
 
         void ISingleProductShop.OnProductLoaded(Product product, ProductVariant[] variants) {
-            _productVariants = variants;
-
-            Title.text = product.title();
-            Description.text = product.description();
-            SetupVariantOptions(variants);
-
-            UpdateDetailsUsingVariant(variants[0]);
-
-            var images = product.images().edges().Select((x) => x.node()).ToArray();
```

Comment:
It would be really cool to have an extension method that can get `Variants()` directly from a `Product()`, that way we arent breaking the graphql model, but also allowing for that more convenient approach.